docs/_config.yml

```diff
@@ -67,4 +67,4 @@ plugins:
   - jekyll-redirect-from
   - jekyll-seo-tag
 
-exclude: ["deploy/includes"]
+exclude: ["deploy/includes", "vendor"]
```

Comment:
What is this?

Reply:
I removed this. I had a config file under doc/bundle directory that made the vendor directory get generated under the docs pkg. I deleted the bundle and vendor dirs form my local and now make docs-serve works without excluding "vendor".